modules/createItem.js

```diff
@@ -38,9 +38,15 @@ export const createTaskElement = (todo) => {
   checkbox.classList.add('larger');
   //   event listener on checkbox
   checkbox.addEventListener('change', () => {
-    statusUpdate(inputText, checkbox, element, todo);
-    localStorage.setItem('arrayOfTasks', JSON.stringify(arrayOfTasks));
+    statusUpdate(arrayOfTasks, inputText, checkbox, element, todo);
   });
+  // keeping the state of checkbox and input as it is:
+  if (todo.completed) {
+    inputText.classList.toggle('completed');
+    inputText.classList.toggle('line-through');
+    checkbox.checked = true;
+  }
+
```

Comment:
- [ ] The changes you have introduced have created a new bug in your app. right now the todos are not getting added to the list. I have tried multiple times but I could not add a todo. kindly consider fixing this bug.
Video Demonstration

https://github.com/SanaTameem/To-do-List/assets/96683843/d5657e39-0f75-406a-a203-1164943ee325